xtractmime/_patterns.py

```diff
@@ -1,102 +1,156 @@
+"""
+Section 6.1, step 1
+https://mimesniff.spec.whatwg.org/commit-snapshots
+/609a3a3c935fbb805b46cf3d90768d695a1dcff2/#matching-an-image-type-pattern
+"""
 IMAGE_PATTERNS = (
-    (b"\x00\x00\x01\x00", b"\xff\xff\xff\xff", "image/x-icon"),  # A Windows Icon signature
-    (b"\x00\x00\x02\x00", b"\xff\xff\xff\xff", "image/x-icon"),  # A Windows Cursor signature
-    (b"BM", b"\xff\xff", "image/bmp"),  # The string "BM", a BMP signature
-    (b"GIF87a", b"\xff\xff\xff\xff\xff\xff", "image/gif"),  # The string "GIF87a", a GIF signature
-    (b"GIF89a", b"\xff\xff\xff\xff\xff\xff", "image/gif"),  # The string "GIF89a", a GIF signature
+    (b"\x00\x00\x01\x00", b"\xff\xff\xff\xff", None, "image/x-icon"),  # A Windows Icon signature
+    (b"\x00\x00\x02\x00", b"\xff\xff\xff\xff", None, "image/x-icon"),  # A Windows Cursor signature
+    (b"BM", b"\xff\xff", None, "image/bmp"),  # The string "BM", a BMP signature
+    (
+        b"GIF87a",
+        b"\xff\xff\xff\xff\xff\xff",
+        None,
+        "image/gif",
+    ),  # The string "GIF87a", a GIF signature
```

Comment:
The position of this type of comment affects many entries of the file that used to be a single line but Black turned into multiple lines. Black is not smart enough to fix that.

Reply:
Maybe it will be better to remove all the comments for each table entry as we are already mentioning the snapshot of the table of patterns